tutorials/local-server-api-db-intro.md

```diff
@@ -0,0 +1,71 @@
+# Local Server
+
+## Overview 
+
+In this module we will bring up the API server and do some basic exploration of the database and make an API call against the server. The goal is to help you gain an understanding of how the back end and data is structured and to start gaining an understanding the tools, scripts and languages we use to aid us in development.
+
+## Tasks
+
+### (LOCAL-SERVER-1) Bring up the local server
+
+./project.rb dev-up
+
+This will bring up a server with API endpoints and a local database.
+
+
+#### Dependencies
```

Comment:
We already have setup steps / prerequisites in our top-level README. I'd rather just link to there so we don't inevitably have these diverging over time.